taskfiles/lint/main.yaml

```
@@ -0,0 +1,47 @@
+version: "3"
+
+includes:
+  utils: "../../exports/taskfiles/utils/utils.yaml"
+  yaml:
+    flatten: true
+    taskfile: "./yaml.yaml"
+
+vars:
+  G_LINT_VENV_DIR: "{{.G_OUTPUT_DIR}}/lint-venv"
+  G_LINT_VENV_CHECKSUM_FILE: "{{.G_OUTPUT_DIR}}/lint#venv.md5"
+
+tasks:
+  check:
+    desc: "Runs all linters to identify any violations."
+    cmds:
+      - task: "check-yaml"
+
+  venv:
+    internal: true
+    vars:
+      CHECKSUM_FILE: "{{.G_LINT_VENV_CHECKSUM_FILE}}"
+      OUTPUT_DIR: "{{.G_LINT_VENV_DIR}}"
+    sources:
+      - "{{.ROOT_DIR}}/lint-requirements.txt"
+      - "{{.ROOT_DIR}}/taskfile.yaml"
+      - "{{.TASKFILE}}"
```

```suggestion
      - "{{.ROOT_DIR}}/exports/taskfiles/**/*.yaml"
      - "{{.ROOT_DIR}}/taskfile.yaml"
      - "{{.TASKFILE_DIR}}/**/*.yaml"
```

The suggestion is so that we track changes to all taskfiles, since they may include each other (e.g., we need to track changes to the utils taskfile).